sklearn/sklearn-template/template/trainer/input_util.py

```diff
@@ -19,12 +19,12 @@
 from sklearn import model_selection
 from google.cloud import storage
 
-from trainer import metadata
+from trainer.metadata import BASE_QUERY
```

Comment:
Everywhere: Suggest using import statements for packages and modules only: http://google.github.io/styleguide/pyguide.html#22-imports